src/components/tasks/TasksContent.tsx

```diff
@@ -35,16 +44,22 @@ export const TasksContent = () => {
 
     if (isLoading) return <p>Loading...</p>;
 
+    const renderTaskList = () => {
+        if (dev === 'true' && tasks.length > 0) {
+            return <TaskList tasks={tasks} />;
+        }
+
+        if (tasksGroupedByStatus[activeTab]) {
+            return <TaskList tasks={tasksGroupedByStatus[activeTab]} />;
+        }
+
+        return <p>{NO_TASKS_FOUND_MESSAGE}</p>;
+    };
+
```

Comment:
sunny Please make this as a separate component and also write test for it

Reply:
You can read about why Not to use functional component more like js functions and why its a good practice to create functional components in a separate file even if they are small

Reply:
Separate components is created in another branch https://github.com/Real-Dev-Squad/website-status/pull/697/commits/b9ce91a524d3b36c81c6b36b92886f9e49ba3778